picoquictest/tls_api_test.c

```diff
@@ -3236,4 +3236,4 @@ int client_error_test()
     }
 
```

Comment:
This is a weird test, dating back to the definition of ping/pong frames in draft 7 or 8. You should leave it alone in the first PR. We probably want to just get rid of it in a separate PR.